checkov/terraform/checks/resource/aws/ElasticsearchNodeToNodeEncryption.py

```diff
@@ -18,21 +18,21 @@ def scan_resource_conf(self, conf):
         :return: <CheckResult>
         """
         if "cluster_config" in conf.keys():
-
-            instance_count = conf["cluster_config"][0]["instance_count"][0]
-            if isinstance(instance_count,int):
-                if instance_count > 1:
-                    if "node_to_node_encryption" in conf.keys():
-                        if conf["node_to_node_encryption"][0]["enabled"][0]:
-                            return CheckResult.PASSED
+            if isinstance(conf["cluster_config"][0], dict):
+                instance_count = conf["cluster_config"][0]["instance_count"][0]
+                if isinstance(instance_count, int):
+                    if instance_count > 1:
+                        if "node_to_node_encryption" in conf.keys():
```

Comment:
@arielkru Please use the `and` operator to simplify this `if-else` case

Reply:
I added only the first row, but no problem i'll edit the rest